tests/data/sentinel2/data.py

```diff
@@ -4,40 +4,106 @@
+                "IMG_DATA": {
+                    "R10m": [
+                        "T26EMU_20220414T110751_AOT_10m.jp2",
+                        "T26EMU_20220414T110751_B02_10m.jp2",
+                        "T26EMU_20220414T110751_B03_10m.jp2",
+                        "T26EMU_20220414T110751_B04_10m.jp2",
+                        "T26EMU_20220414T110751_B08_10m.jp2",
+                        "T26EMU_20220414T110751_TCI_10m.jp2",
+                        "T26EMU_20220414T110751_WVP_10m.jp2",
+                    ],
+                    "R20m": [
+                        "T26EMU_20220414T110751_AOT_20m.jp2",
+                        "T26EMU_20220414T110751_B01_20m.jp2",
+                        "T26EMU_20220414T110751_B02_20m.jp2",
+                        "T26EMU_20220414T110751_B03_20m.jp2",
+                        "T26EMU_20220414T110751_B04_20m.jp2",
+                        "T26EMU_20220414T110751_B05_20m.jp2",
+                        "T26EMU_20220414T110751_B06_20m.jp2",
+                        "T26EMU_20220414T110751_B07_20m.jp2",
+                        "T26EMU_20220414T110751_B11_20m.jp2",
+                        "T26EMU_20220414T110751_B12_20m.jp2",
+                        "T26EMU_20220414T110751_B8A_20m.jp2",
+                        "T26EMU_20220414T110751_SCL_20m.jp2",
+                        "T26EMU_20220414T110751_TCI_20m.jp2",
+                        "T26EMU_20220414T110751_WVP_20m.jp2",
+                    ],
+                    "R60m": [
+                        "T26EMU_20220414T110751_AOT_60m.jp2",
+                        "T26EMU_20220414T110751_B01_60m.jp2",
+                        "T26EMU_20220414T110751_B02_60m.jp2",
+                        "T26EMU_20220414T110751_B03_60m.jp2",
+                        "T26EMU_20220414T110751_B04_60m.jp2",
+                        "T26EMU_20220414T110751_B05_60m.jp2",
+                        "T26EMU_20220414T110751_B06_60m.jp2",
+                        "T26EMU_20220414T110751_B07_60m.jp2",
+                        "T26EMU_20220414T110751_B09_60m.jp2",
+                        "T26EMU_20220414T110751_B11_60m.jp2",
+                        "T26EMU_20220414T110751_B12_60m.jp2",
+                        "T26EMU_20220414T110751_B8A_60m.jp2",
+                        "T26EMU_20220414T110751_SCL_60m.jp2",
+                        "T26EMU_20220414T110751_TCI_60m.jp2",
+                        "T26EMU_20220414T110751_WVP_60m.jp2",
+                    ],
+                }
+            }
+        }
+    },
+}
 
 
 def create_file(path: str, dtype: str, num_channels: int) -> None:
     profile = {}
-    profile["driver"] = "GTiff"
+    profile["driver"] = "JP2OpenJPEG"
     profile["dtype"] = dtype
     profile["count"] = num_channels
-    profile["crs"] = "epsg:4326"
-    profile["transform"] = rasterio.transform.from_bounds(0, 0, 1, 1, 1, 1)
+    profile["crs"] = CRS.from_epsg(32616)
+    profile["transform"] = Affine(10.0, 0.0, 399960.0, 0.0, -10.0, 4500000.0)
```

Comment:
This will make every file have 10m/px resolution. Is this desired?

Reply:
Fixed